pkg/agent/cniserver/server_linux_test.go

```diff
@@ -303,6 +303,8 @@ func TestCmdAdd(t *testing.T) {
 			requestMsg, hostInterfaceName := createCNIRequestAndInterfaceName(t, testPodNameA, tc.cniType, ipamResult, tc.ipamType, true)
 			testIfaceConfigurator.hostIfaceName = hostInterfaceName
 			cniserver.podConfigurator.ifConfigurator = testIfaceConfigurator
+			kubeClient := fakeclientset.NewClientset(pod1, pod2, pod3)
+			cniserver.kubeClient = kubeClient
```

Comment:
No unit test case to cover the new annotation update?

Reply:
updated